src/render.ts

```diff
@@ -172,7 +172,16 @@ export function render(s: State): void {
 }
 
 export function updateBounds(s: State): void {
+  const bounds = s.dom.elements.wrap.getBoundingClientRect();
```

Comment:
There's potentially some duplicated work between this `getBoundingClientRect()` here and using the memoized `s.dom.bounds()` below, another call to `getBoundingClientRect()`. Maybe this can be simplified?

Reply:
Hm, we could use the calculated width but we also need the values for `left` and `top` and there's no guarantee that the board is top-left aligned inside the wrapper. In fact, as I think I mentioned on Discord before, it might even be worth considering switching the board on Lichess to be right-aligned in order to avoid a gap between the board and the eval gauge.

I guess we could decide to just assume the alignment Lichess is using though?

Reply:
Ah, I see. No reason to hold back the patch for this decision, so let's postpone it.